visualization/app/codeCharta/util/metric/metricDescriptions.ts

```diff
@@ -0,0 +1,20 @@
+/*
```

Comment:
We could export the metricDescriptions constant itself instead of a getter function.
This would avoid the getter calls in the legend component.